src/components/TodoCalendar.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Calendar from "react-calendar";
+import styled from "styled-components";
+import "react-calendar/dist/Calendar.css";
+import ProgressBar from "./ProgressBar";
+import moment from "moment";
+
+// 캘린더
+const TodoCalendar = ({ setDate, todos, progress }) => {
+  const handleDateChange = (selectedDate) => {
+    const year = selectedDate.getFullYear();
+    const month = String(selectedDate.getMonth() + 1).padStart(2, "0"); // YYYY-MM-DD 형식과 일치시키기 위해 한 자리 수면 앞에 0 붙임
+    const day = String(selectedDate.getDate()).padStart(2, "0"); // YYYY-MM-DD 형식과 일치시키기 위해 한 자리 수면 앞에 0 붙임
+    const formattedDate = `${year}-${month}-${day}`;
```

Comment:
moment에 format도 있습니당~~~~
```suggestion
    const formattedDate = moment(selectedDate).format('YYYY-MM-DD');
```

Reply:
앗... 바닐라js 코드를 리팩토링 때 거의 그대로 쓴 다음에 달력 넣었더니 이걸 생각 못 했네요... 알려주신 것처럼 moment 쓰니까 코드가 엄청 간단해지는..😂 담엔 꼭 쓸게여